.github/kokoro/presubmit-install.cfg

```diff
@@ -0,0 +1,38 @@
+# Format: //devtools/kokoro/config/proto/build.proto
```

Comment:
I don't think we want a presubmit install?

Reply:
Problem is that having the make install target in xc7 is not entirely correct.
We may want to have in the install directory the various databases/rr_graphs/etc. of potentially other families, such as ice40 for instance.
It would not be consistent to have the install target in the xc7 directory.

Reply:
@mithro actually I am confused, what is the difference between presubmit and continuous?

Reply:
Presubmit -- run in pull requests.
Continuous -- run after the stuff has been merged to master branch.

Reply:
We want both to ensure that PR's don't break it once submitted.